barriers/views/search.py

```diff
@@ -124,6 +124,18 @@ def get_saved_search_context_data(self, form):
         context_data = {}
         saved_search = self.get_saved_search(form)
         if saved_search:
+            # If user has clicked the "update saved search" button, get the updated
+            # saved_search and continue operation using the new version
+            if "update_search" in self.request.GET and form.cleaned_data.get(
+                "search_id"
+            ):
+                self.client.saved_searches.patch(
+                    id=str(form.cleaned_data.get("search_id")),
```

Comment:
do we need to wrap it in str()?

```suggestion
                    id=form.cleaned_data.get("search_id"),
```

Reply:
I've copied what it was doing below up to here, but i'll check if removing it breaks anything

Reply:
Yup, seems fine, committing